src/endpoints/user.rs

```diff
@@ -7,20 +7,31 @@ use chrono::{DateTime, Utc};
 /// Gets information about a user
 /// https://api.cloudflare.com/#user-user-details
 
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq, Eq)]
+pub struct Organization {
+    id: String,
+    name: String,
+    status: String,
+    permissions: Vec<String>,
+    roles: Vec<String>,
+}
+
 #[derive(Deserialize, Serialize, Debug, Clone, PartialEq, Eq)]
 pub struct UserDetails {
-    pub id: String,
-    pub email: String,
-    pub first_name: String,
-    pub last_name: String,
+    pub organizations: Vec<Organization>,
+    pub betas: Vec<String>,
+    pub telephone: Option<String>,
+    pub zipcode: Option<String>,
+    pub last_name: Option<String>,
+    pub modified_on: DateTime<Utc>,
     pub username: String,
-    pub telephone: String,
-    pub contry: String,
-    pub zipcode: String,
     pub created_on: DateTime<Utc>,
-    pub modified_on: DateTime<Utc>,
```

Comment:
The API docs say there's still a `modified_on` field, are they wrong or is there some other reason for deleting them?